Doc/library/unittest.mock.rst

```diff
@@ -2831,3 +2831,122 @@ Sealing mocks
         >>> mock.not_submock.attribute2  # This won't raise.
 
     .. versionadded:: 3.7
+
+
+Order of precedence of side_effect, return_value and wraps
+----------------------------------------------------------
```

Comment:
This would be better, providing it renders okay.
```suggestion
Order of precedence of :attr:`side_effect`, :attr:`return_value` and *wraps*
----------------------------------------------------------------------------
```